nix/internal/darwin-terminal-init.sh

```diff
@@ -9,20 +9,23 @@ export PATH
 reset
 
 color_bold=$'\e[1m'
+color_underline=$'\e[4m'
 color_reset=$'\e[0m'
 
 cat <<EOF
 
-  Hello!
+  Welcome!
 
-  This terminal has been set up to allow easy running of ‘${color_bold}blockfrost-platform${color_reset}’.
+  This terminal is set up for easy execution of the ‘${color_bold}blockfrost-platform${color_reset}’.
 
-  Try typing in one of:
+  Run one of the following commands:
 
-    ${color_bold}blockfrost-platform --help${color_reset}
-    ${color_bold}blockfrost-platform --init${color_reset}
+   For setup: ${color_bold}blockfrost-platform --help${color_reset}
+   For help:  ${color_bold}blockfrost-platform --init${color_reset}
```

Comment:
Oh, wait, it's the other way around! Help should be `--help`, and setup should be `--init`.